src/ir/predicate_test.cc

```diff
@@ -116,94 +116,78 @@ TEST_P(PredicateEqTest, PredicateEqTest) {
+    std::make_unique<TagPresence>("tag2"), std::make_unique<TagPresence>("tag3"));
+
+static const Or kOrTag1Tag2_2(
+    std::make_unique<TagPresence>("tag1"), std::make_unique<TagPresence>("tag2"));
+static const Or kOrTag1Tag3_2(
+    std::make_unique<TagPresence>("tag1"), std::make_unique<TagPresence>("tag3"));
+static const Or kOrTag2Tag3_2(
+    std::make_unique<TagPresence>("tag2"), std::make_unique<TagPresence>("tag3"));
+
+static const Implies kImpliesTag1Tag2(
+    std::make_unique<TagPresence>("tag1"), std::make_unique<TagPresence>("tag2"));
+static const Implies kImpliesTag1Tag2_2(
+    std::make_unique<TagPresence>("tag1"), std::make_unique<TagPresence>("tag2"));
+static const Implies kImpliesTag1Tag3(
+    std::make_unique<TagPresence>("tag1"), std::make_unique<TagPresence>("tag3"));
+static const Implies kImpliesTag1Tag3_2(
+    std::make_unique<TagPresence>("tag1"), std::make_unique<TagPresence>("tag3"));
+static const Implies kImpliesTag2Tag3(
+    std::make_unique<TagPresence>("tag2"), std::make_unique<TagPresence>("tag3"));
+static const Implies kImpliesTag2Tag3_2(
+    std::make_unique<TagPresence>("tag2"), std::make_unique<TagPresence>("tag3"));
+
+static const Not kNotTag1(std::make_unique<TagPresence>("tag1"));
+static const Not kNotTag1_2(std::make_unique<TagPresence>("tag1"));
+static const Not kNotTag2(std::make_unique<TagPresence>("tag2"));
+static const Not kNotTag2_2(std::make_unique<TagPresence>("tag2"));
+static const Not kNotTag3(std::make_unique<TagPresence>("tag3"));
+static const Not kNotTag3_2(std::make_unique<TagPresence>("tag3"));
 
 static const std::pair<const Predicate *, uint64_t> example_predicates[] = {
-    { kTag1Present.predicate(), 0},
-    { kTag1Present2.predicate(), 0},
-    { kTag2Present.predicate(), 1},
-    { kTag2Present2.predicate(), 1},
-    { kTag3Present.predicate(), 2},
-    { kTag3Present2.predicate(), 2},
-    { kAndTag1Tag2.predicate(), 3},
-    { kAndTag1Tag2_2.predicate(), 3},
-    { kAndTag1Tag3.predicate(), 4},
-    { kAndTag1Tag3_2.predicate(), 4},
-    { kAndTag2Tag3.predicate(), 5},
-    { kAndTag2Tag3_2.predicate(), 5},
-    { kOrTag1Tag2.predicate(), 6},
-    { kOrTag1Tag2_2.predicate(), 6},
-    { kOrTag1Tag3.predicate(), 7},
-    { kOrTag1Tag3_2.predicate(), 7},
-    { kOrTag2Tag3.predicate(), 8},
-    { kOrTag2Tag3_2.predicate(), 8},
-    { kImpliesTag1Tag2.predicate(), 9},
-    { kImpliesTag1Tag2_2.predicate(), 9},
-    { kImpliesTag1Tag3.predicate(), 10},
-    { kImpliesTag1Tag3_2.predicate(), 10},
-    { kImpliesTag2Tag3.predicate(), 11},
-    { kImpliesTag2Tag3_2.predicate(), 11},
-    { kNotTag1.predicate(), 12},
-    { kNotTag1_2.predicate(), 12},
-    { kNotTag2.predicate(), 13},
-    { kNotTag2_2.predicate(), 13},
-    { kNotTag3.predicate(), 14},
-    { kNotTag3_2.predicate(), 14}
-};
+    {&kTag1Present, 0},      {&kTag1Present2, 0},
+    {&kTag2Present, 1},      {&kTag2Present2, 1},
+    {&kTag3Present, 2},      {&kTag3Present2, 2},
```

Comment:
I think this formatting is really readable and nice!

Reply:
Well, `clang-format` did it for me. :)